extensions/notebook/src/dialog/managePackages/installedPackagesTab.ts

```diff
@@ -92,7 +92,7 @@ export class InstalledPackagesTab {
 						}
 					],
 					data: [[]],
-					height: '600px',
+					height: '500px',
```

Comment:
out of curiosity, why did changing the height remove the horizontal scrollbar?

Reply:
Lucy can confirm - but if I had to guess it's because the extra height caused the vertical scrollbar to appear, and then the extra space that took caused it to also need to show the horizontal scrollbar.

Reply:
Yep Charles is correct